### DISCUSSION TITLE
Application for github open-source fund

### DISCUSSION THREAD

**Discussion by thunder-007** [encode/django-rest-framework#9611] [Ideas & Suggestions]
Applying for this https://github.blog/news-insights/company-news/announcing-github-secure-open-source-fund/ will help the project I tried applying for this project myself but couldn't find all the details. Here is the form https://forms.gle/PWGe69CM3h4ZvvGZ9

**Comment by auvipy** [comment]
I think it should be the projects maintainers who should apply for this.
